wiki/Tournaments/THOM_CUP/1/en.md

```diff
@@ -0,0 +1,192 @@
+---
+tags:
+- THOM
+- THOM CUP
+---
+THOM CUP #1
+======================
+
+![THOM CUP Logo](logo.png)
+
+## Preface
+
+**THOM CUP** is a one-on-one euro-american osu!mania tournament. It is the 1st installment of the THOM CUP. As the title indicates, this tournament is for the osu!mania game mode. More specifically, this tournament is an osu!mania 7-key tournament.
```

Comment:
As the title indicates, this tournament is for the osu!mania game mode. 

this sentence is unneeded i think